src/SFA.DAS.AssessorService.Infrastructure/ApiClients/QnA/IQnaApiClient.cs

```diff
@@ -7,17 +7,26 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 
-namespace SFA.DAS.AssessorService.Application.Api.Client.QnA
+namespace SFA.DAS.AssessorService.Infrastructure.ApiClients.QnA
 {
     public interface IQnaApiClient
     {
         Task<StartApplicationResponse> StartApplications(StartApplicationRequest startAppRequest);
-        Task<ApplicationData> GetApplicationData(Guid applicationId);
+        Task<T> GetApplicationData<T>(Guid applicationId);
```

Comment:
@ChrisJWoodcock whats the reason for making `GetApplicationData` a genric type ?

Reply:
I have synced the implementation of the QnAClient between admin and assessor in a follow up ticket the QnAClient which now has two separate but identical implementation will be migrated to the QnA API repo, and used in both place via a nuget, however when that method is called it serializes the dictionary to a known type which will be different from different callers - from admin and assessor it will be the same type when talking about assessor qna data but from RoATP it would be the RoATP definition of qna data